integration-tests/spring-data-jpa/src/main/java/io/quarkus/it/spring/data/jpa/PostResource.java

```diff
@@ -104,6 +104,27 @@ public Post byBypassTrue() {
         return postCommentRepository.findByPostId(id);
     }
 
+    @GET
+    @Produces("application/json")
+    @Path("/mandatory/{id}")
+    public Post findMandatoryByPostId(@PathParam("id") Long id) {
+        return postRepository.findMandatoryById(id);
+    }
+
+    @GET
+    @Produces("application/json")
+    @Path("/doNothing")
+    public void doNothing() {
+        postRepository.doNothing();
+    }
+
+    @GET
+    @Produces("application/json")
+    @Path("/attribute/{attribute}/text/{text}")
+    public void findByAttributeContainsText(@PathParam("attribute") String attribute, @PathParam("text") String text) {
+        //        postRepository.findByAttributeContainsText(attribute, text);
```

Comment:
Is there a reason this is commented out?

Reply:
removed. it should not be there